src/odin/maneuversbuilder.cc

```diff
@@ -2701,6 +2701,9 @@ bool ManeuversBuilder::AreRoundaboutsProcessable(const TripLeg_TravelMode travel
 }
 
 void ManeuversBuilder::ProcessRoundabouts(std::list<Maneuver>& maneuvers) {
+  std::vector<midgard::PointLL> shape =
```

Comment:
originally i was thinking about assigning in triplegbuilder so we do not have to decode shape again - but I do not think that is possible